mixer/pkg/il/interpreter/interpreterRun.go

```diff
@@ -123,7 +123,7 @@ func (in *Interpreter) run(fn *il.Function, bag attribute.Bag, step bool) (Resul
 			if sp < 2 {
 				goto STACK_UNDERFLOW
 			}
-			sp = sp - 2
+			sp -= 2
```

Comment:
This file is auto-generated...

Reply:
Ah. It should have a .gen.go extension then. I'll fix that.

Reply:
OK, I updated the template and renamed the output file to .gen.go.